openapi-cli/src/main/java/io/ballerina/openapi/cmd/utils/CodegenUtils.java

```diff
@@ -36,6 +39,24 @@ public static Path getImplPath(String pkg, Path srcPath) {
         return (pkg == null || pkg.isEmpty()) ? srcPath : srcPath.getParent();
     }
 
+    /**
+     * Resolves path to write generated implementation source files.
+     *
+     * @param fileName     name of the file which contains the content of prefix
+     */
+    public static void setPrefixContent(String fileName) throws IOException {
+        prefixContent = "";
+        if (fileName != null && !fileName.isBlank()) {
+            Path prefixFilePath = Paths.get((new File(fileName).getCanonicalPath()));
```

Comment:
Can this file be at any place in the disk? Or should it be within the project? Is it file name or file path that we need?

Reply:
This file can be at any place in the disk. We need to give the file path, not the name. Updated the parameter name to `licenseFilePath`. Please review.